bin/utils/convert-argv.js

```diff
@@ -104,14 +104,29 @@ module.exports = function(...args) {
 
 		const requireConfig = function requireConfig(configPath) {
 			let options = (function WEBPACK_OPTIONS() {
+				const resolvedPath = path.resolve(process.cwd(), configPath);
 				if (argv.configRegister && argv.configRegister.length) {
 					module.paths.unshift(path.resolve(process.cwd(), "node_modules"), process.cwd());
 					argv.configRegister.forEach(dep => {
 						require(dep);
 					});
-					return require(path.resolve(process.cwd(), configPath));
+					return require(resolvedPath);
 				} else {
-					return require(path.resolve(process.cwd(), configPath));
+					try {
+						return require(resolvedPath);
+					} catch (err) {
+						if (err.stack.indexOf("ERR_REQUIRE_ESM") >= 0) {
```

Comment:
```suggestion
						if (err.stack.includes("ERR_REQUIRE_ESM")) {
```

Reply:
@evenstensberg it seems there're plenty of other instances where `indexOf` is used to check for existence instead of `includes()`. Would you accept a PR for this one?

Reply:
no, this is not a PR to make the code prettier, it's a PR to support LTS while we make things work well at `next`.